### DISCUSSION TITLE
JavaScript 基础知识  2.2代码结构    分号部分   错误的例子讲解有错误

### DISCUSSION THREAD

**Discussion by soulcreator** [javascript-tutorial/zh.javascript.info#1045] [General]
![image](https://user-images.githubusercontent.com/42525568/168459043-0cf0bb59-199d-478c-a96c-dcf16f3b3985.png)

这段例子是可以弹出[1,2]中的值，去掉换行才会报错

**Comment by songhn233** [comment]
<img width="409" alt="image" src="https://user-images.githubusercontent.com/47357585/168463372-906c6675-f772-49e0-b16d-06f79f19ffb9.png">

这个应该是没有问题的，请提供更多信息
